inst/extdata/install-packages.R

```diff
@@ -0,0 +1,177 @@
+# Message strings used in the setup script
+packageInstallationMessages <- list(
+  installRTools = "Install Rtools and / or specify a path to an existing installation, then run updateEnvironment(rtoolsPath = path)",
+  RToolsNotFound = function(rToolsPath) {
+    paste0("Rtools not found at ", rToolsPath, ", cannot continue")
+  },
+  PKSimLoadFails = "PK-Sim fails to load. The installation might be incompatible with your current version of PK-Sim"
+)
+
+# Download paths of released package versions
+.releasePaths <- list(
+  ospsuite.utils = "https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils/releases/download/v1.3.17/ospsuite.utils_1.3.17.zip",
+  tlf = "https://github.com/Open-Systems-Pharmacology/TLF-Library/releases/download/v1.4.89/tlf_1.4.89.zip",
```

Comment:
let's use .tar.gz instead of zip where possible.
a) packages which are not OS-specific (like tlf or ospsuite.utils) can be installed under Linux then without problems
b) tests can be installed on demand

Regarding the last point: I would convert this script into **function** and provide optional argument for the installation of the package tests